packages/forms-web-app/src/dynamic-forms/validator/validOptionValidator.js

```diff
@@ -0,0 +1,14 @@
+const { body } = require('express-validator');
+const BaseValidator = require('./baseValidator.js');
+
+class ValidOptionValidator extends BaseValidator {
+	validate(questionObj, errorMessage) {
+		return body(questionObj.fieldName)
+			.custom((value) => {
+				if (!questionObj.options.includes(value)) throw new Error();
```

Comment:
As each option in the options array is an object with the `value` property being the one we're interested in, I think, rather than `includes()` we'd have to use `some()` like this:
`if (!questionObj.options.some(option => option.value == value)) throw new Error();`

Reply:
I have assumed option.value and value are of the same type, and have used strict equality, i.e.'===